fairgame.js

```diff
@@ -390,8 +390,29 @@ function setLadderRows() {
     clientData.ladderPadding = qolOptions.expandedLadder.size / 2;
 }
 
+
+function expandLadder() {
+    if(document.getElementsByClassName("ladder-container").length > 0) {
+        return;
+    }
+    var input = 10000;
+    qolOptions.expandedLadder.size = input;
+    clientData.ladderPadding = qolOptions.expandedLadder.size / 2;
+    var ladder = document.querySelector(".caption-top");
+    var ladderParent = ladder.parentElement;
+    var ladderContainer = document.createElement("div");
+    ladderContainer.className = "ladder-container";
+    ladderContainer.style.width = "100%";
+    ladderContainer.style.height = "64vh";
+    ladderContainer.style.overflow = "auto";
+    ladderContainer.style.border = "gray solid 2px";
+    ladderParent.replaceChild(ladderContainer, ladder);
+    ladderContainer.appendChild(ladder);
+}
+
 addJS_Node(secondsToHms);
 addJS_Node(solveQuadratic);
+addJS_Node(expandLadder);
```

Comment:
Can you add an option to call expandLadder() automatically?
if (qolOptions.scrollableLadder) {expandLadder()}

Reply:
Would need to add that config at the top as well then